api/src/opentrons/protocols/context/implementation_factory.py

```diff
@@ -0,0 +1,114 @@
+"""Factory functions to create context implementation classes."""
```

Comment:
```suggestion
"""Factory functions to create concrete contexts."""
```

Would this be clearer?